### PR TITLE
TASK: Refresh NodeTree if Workspace is switched

### PR DESCRIPTION
**Pull request by johannessteu**
If you switch the workspace the NodeTree will be reloaded.

NEOS-1753 #close

### CODE REVIEW

**Comment by dfeyer** [comment]
:+1:

**Comment by aertmann** [comment]
:+1: by reading

File: TYPO3.Neos/Resources/Public/JavaScript/Content/Components/TargetWorkspaceController.js [hunk 1]
```diff
@@ -156,6 +156,8 @@ define(
 				HttpRestClient.updateResource('neos-service-workspaces', this.get('userWorkspace.name'), options).then(function(result) {
 					that._loadConfiguration();
 					ContentModule.reloadPage();
+					EventDispatcher.trigger('nodesInvalidated');
+					EventDispatcher.trigger('contentChanged');
```
Comment (aertmann) [inline TYPO3.Neos/Resources/Public/JavaScript/Content/Components/TargetWorkspaceController.js]: actually seems like two events for the same thing?

Comment (johannessteu) [inline TYPO3.Neos/Resources/Public/JavaScript/Content/Components/TargetWorkspaceController.js]: Nope, `nodesInvalidated` will trigger the nodetree to reload, `contentChanged` the structureTree. None of those events triggered the structure and the node tree. Or is there another event that could be used @aertmann ?

**Comment by kdambekalns** [comment]
:+1: - works like a charm

**Pull request merged**